subsys/bluetooth/host/Kconfig.l2cap

```
@@ -32,6 +32,7 @@ config BT_L2CAP_TX_MTU
 	default 253 if BT_BREDR
 	default 69 if BT_MESH_GATT
 	default 65 if BT_SMP
+	default 64 if BT_AUDIO_UNICAST_SERVER || BT_AUDIO_UNICAST_CLIENT || BT_AUDIO_BROADCAST_SINK || BT_BASS || BT_BASS_CLIENT
```

```suggestion
	default 64 if BT_AUDIO_UNICAST
```

I don't think we need to support a high MTU for e.g. the broadcast roles, right?

---

The BAP spec says:
"The Broadcast Sink shall support a minimum ATT_MTU of 64 octets for one Unenhanced ATT bearer or
for at least one Enhanced ATT bearer if the Broadcast Sink supports Enhanced ATT bearers."

:confused:

---

I see. That's seemingly because it shall instantiate PACS (created https://github.com/zephyrproject-rtos/zephyr/issues/43788 for this). 

Looks like this is the right way of setting it then